<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetTransitions</class>
 <widget class="QWidget" name="SetTransitions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список переходов</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>560</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameState">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="toStates">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Список переходов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okayButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>560</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="numberState">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>18</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Номер состояния</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addStateButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить состояние</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>560</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить список состояний</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
